pkg/kubelet/cm/cpumanager/cpu_assignment_test.go

```diff
@@ -509,6 +509,7 @@ func TestCPUAccumulatorTake(t *testing.T) {
 type takeByTopologyTestCase struct {
 	description   string
 	topo          *topology.CPUTopology
+	opts          StaticPolicyOptions
```

Comment:
do we need this option struct anymore? (same for the rest of changes in this file)

Reply:
Some other tests inherit this common structure but I double checked and noticed that there's no references after last update. nice catch! will remove the unnecessary codes